docuilib/src/components/pageComponents/SectionContent.tsx

```diff
@@ -13,27 +13,22 @@ export const SectionContent = ({section, component}) => {
         {_.map(content, (item, index: number) => {
           const isLast = index === content.length - 1;
 
-          return <div style={{marginBottom: isLast ? 0 : 40, border: '1px solid #F8F9FA'}}><ContentItem item={item} componentName={component.name}/></div>;
+          return <div style={{marginBottom: isLast ? 0 : 40, border: '1px solid #F8F9FA', background: item.background}}><ContentItem item={item} componentName={component.name}/></div>;
```

Comment:
Why use `background` and not `backgroundColor`?

Reply:
It's a div so it's the same